docs/development/releasing.rst

```diff
@@ -283,6 +283,22 @@ packages that use the full bugfix/maintenance branch approach.)
    in the ``ci-helpers`` repository once the ``conda`` packages became
    available.
 
+#. Upload the release to Zenodo. This has to be done manually since the
+   Zenodo/GitHub integration relies on making releases on GitHub, which we
+   don't do. So for the Astropy core package, log in to
```

Comment:
However pushing a tag creates a github release. 
@keflavich  can confirm or refute whether zenodo is done automatically for astroquery, but I suspect it's.

Reply:
No, we don't have anything automatic set up with zenodo.  I didn't know you could do that.  Tell me more!

Reply:
Hmm, it's listed as an integration. But maybe then it's not active for the same reason Tom mentions here in the paragraph.

Reply:
huh, no, I think you're right... I don't remember doing this:
https://zenodo.org/record/2002585
so it must just be triggered!  Neat!  (note the versions on the right side of that page)

I don't think our new approach of release-per-PR is triggering zenodo releases though

Reply:
No, that doesn't trigger it as there are no tags and thus no "releases" on github. Which is I think more favourable than having a zenodo for each PR merge.

Reply:
Is there a specific reason we do not publish releases on github?

Reply:
We do. Pushing a tag up to github creates a "release". https://github.com/astropy/astropy/releases

Reply:
Then do we need to do the manual stuff for zenodo? For my `baseband` package, the integration does work.

Reply:
One downside of the integration is that the Zenodo entry won't include a proper sdist of the package but instead a tar file of the repo. In any case I'm not sure if it's possible to switch to the integration now that we've started manually - I need to check.